test/pom.xml

```diff
@@ -229,7 +229,6 @@ THE SOFTWARE.
         <artifactId>maven-surefire-plugin</artifactId>
         <!-- version specified in grandparent pom -->
         <configuration>
-          <argLine>${jacocoSurefireArgs} -Xmx1g</argLine>
```

Comment:
Is there a way to define a property in the `jdk-9-and-above` profile that defaults to empty which could be inserted here, so we do not need to duplicate `${jacocoSurefireArgs} -Xmx1g`? I guess it not worth the bother given that this is a temporary situation anyway.

Reply:
I do not think that reducing duplication is always a worthy goal, especially if it comes at the cost of additional complexity.